frontend/components/landing/LandingContent.vue

```diff
@@ -10,13 +10,13 @@
       <img
         v-if="$colorMode.value == 'light'"
         class="mb-4 h-36 sm:h-44 lg:hidden"
-        :src="imageURL + '_light.png'"
+        :src="imgUrl + '_light.png'"
```

Comment:
I think at some point it would be good to refactor conditional image fetching like this into a composable, but that's probably out of scope for this PR

Reply:
Do you want to make an issue for this, @cquinn540?